webroot/css/toolbar.css

```diff
@@ -110,10 +117,14 @@ p {
 	vertical-align: middle;
 }
 
+#panel-button-right {
```

Comment:
@markstory it would help,  if stickler-ci would provide a resource link, which explains why.